<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_matlist</class>
 <widget class="QWidget" name="Form_matlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_projekt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_projektauswahl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filter">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_filter">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xxx Zeilen:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>521</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_matpos">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_check_all_pos">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>alle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_check_no_pos">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>keine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pos_new">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>neu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pos_edit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pos_delete">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>entf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pos_copy">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>kopieren</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pos_import">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>importieren</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
